searx/templates/simple/search.html

```diff
@@ -1,6 +1,8 @@
 <form id="search" method="{{ method or 'POST' }}" action="{{ url_for('search') }}">
   <a id="search_logo" href="{{ url_for('index') }}">
-    <img class="search_logo_img" src="{{ url_for('static', filename='img/searxng-wordmark.svg') }}">
+    <div class="search_logo_img">
+      {{ insert_xml('img/searxng-wordmark.svg') }}
+    </div>
```

Comment:
this leads into  ..

![grafik](https://user-images.githubusercontent.com/554536/140546001-fabc20b7-e5ef-4867-8bce-7eca6fe39608.png)

 and I do not know how I can set a fixed size for the `<div>` container or how the can set a fixed box-size for the  _inline SVG_ ..

I'm not an CSS expert and I haven't understand the CSS box model really.